poetry.lock

```diff
@@ -2094,51 +2094,75 @@ files = [
 
 [[package]]
 name = "pandas"
-version = "1.5.3"
+version = "2.2.2"
```

Comment:
Here, pandas is at v2. `poetry.lock` is what is being used when developing locally. It is not put on `pypi` for the dependent package to install.